Docs: i18n fixes per KR: LOC/23Q2-KR1 (3rd batch of files) (#364)
vtexdocs/dev-portal-content: docs/guides/Getting-Started/getting-started/getting-started-list-of-rest-apis.md

| [CMS API](https://developers.vtex.com/docs/api-reference/cms-api)                                            | Manages VTEX CMS via API.                                                                                                                                                  |
| [Customer Credit API](https://developers.vtex.com/docs/api-reference/customer-credit-api#overview)           | Enables credit payments on your store and controls invoices and credit limits.                                                                                             |
| [Gift Card API](https://developers.vtex.com/docs/api-reference/giftcard-api#overview)                        | Retrieves and updates gift cards from our native gift card system.                                                                                                         |
| [Gift Card Hub API](https://developers.vtex.com/docs/api-reference/giftcard-hub-api#overview)                | Interacts with all gift card providers in a store from a single place.                                                                                                     |

> | [Gift Card Provider Protocol API](https://developers.vtex.com/docs/api-reference/giftcard-provider-protocol) | Set of definitions to become a gift card provider on VTEX.                                                                                                                 |

Reviewer: ```suggestion
| [Gift Card Provider Protocol API](https://developers.vtex.com/docs/api-reference/giftcard-provider-protocol) | Defines the requirements to become a gift card provider on VTEX.                                                                                                                 |
```

Reviewer: Sugestão por conta do paralelismo da tabela (todas as descrições começam com um verbo)
